packages/lib/server/repository/organization.ts

```diff
@@ -170,6 +170,7 @@ export class OrganizationRepository {
     const orgs = await prisma.team.findMany({
       where: {
         isOrganization: true,
+        isPlatform: false,
```

Comment:
dont we need to update platform libraries for this change to be taken into account ?